marimo/_ast/app.py

```diff
@@ -238,6 +240,22 @@ async def _run_async(self) -> tuple[Sequence[Any], dict[str, Any]]:
     def run(self) -> tuple[Sequence[Any], dict[str, Any]]:
         return asyncio.run(self._run_async())
 
+    async def _run_cell_async(
+        self, cell: Cell, kwargs: dict[str, Any]
+    ) -> tuple[Any, _Namespace]:
+        self._maybe_initialize()
+        output, defs = await self._runner.run_cell_async(
+            cell._cell.cell_id, kwargs
+        )
+        return output, _Namespace(defs)
```

Comment:
i like the api for what is returned. its simple for the 99% case (just needing `output`) but `_Namespace` can be extended in future if we need to pass more info